src/e2e/scala/temple/SimpleE2ETestData.scala

```diff
@@ -27,19 +27,19 @@ object SimpleE2ETestData {
   val dockerfile: String =
     """FROM golang:1.13.7-alpine
       |
-      |WORKDIR /templeuser
+      |WORKDIR /temple-user
       |
       |COPY go.mod go.sum ./
       |
       |RUN ["go", "mod", "download"]
       |
       |COPY . .
       |
-      |COPY config.json /etc/templeuser-service
+      |COPY config.json /etc/temple-user-service
```

Comment:
Think your branch might be outdated - this should have a trailing slash from #224